### PR TITLE
Move multi-GPU dlpack test to a separate L0 test

### PR DESCRIPTION
**Pull request by Tabrizian**
It takes a lot of time to find a multi-GPU machine in CI. This should enable us to iterate faster.

### CODE REVIEW

File: qa/L0_dlpack_multi_gpu/test.sh
Comment (oandreeva-nv) [inline qa/L0_dlpack_multi_gpu/test.sh:1]: I think copyright should be `2023`

**Comment by oandreeva-nv** [comment]
LGTM, thank you for this PR! Just one change with copyright year and I'll approve

**Review by dyastremsky — APPROVED** [review]

**Review by oandreeva-nv — APPROVED** [review]

**Review by rmccorm4 — DISMISSED** [review]
Nice optimization 🚀

**Review by oandreeva-nv — APPROVED** [review]
Thanks!

**Review by dyastremsky — APPROVED** [review]

**Pull request merged**